contracts/pool-manager/sim/lp_fee.py

```
@@ -0,0 +1,178 @@
+            fees.append(_dynamic_fee_i * difference / FEE_DENOMINATOR)
+            new_balances[i] -= fees[i]
+        D1 = get_D(new_balances, amp)
+        mint_amount = total_supply * (D1 - D0) / D0
+        print("fees: ", fees)
+
+    return mint_amount
+
+FEE_DENOMINATOR = 100
+
+def _dynamic_fee(xpi, xpj, _fee):
+    _offpeg_fee_multiplier: uint256 = 1000
```

Ruff (0.8.2)

Undefined name `uint256`

(F821)